docs/sites.yml

```diff
@@ -9859,3 +9859,16 @@
     - Agency
   built_by: App Design
   built_by_url: "https://appdesign.dev/"
+  - title: Guillermo Gómez-Peña
```

Comment:
```suggestion
- title: Guillermo Gómez-Peña
```

Reply:
Sorry I missed that, and thanks for the suggestion!